packages/eslint-config-eventbrite-react/rules/react-a11y.js

```diff
@@ -4,6 +4,55 @@ module.exports = {
     // View link below for docs on react a11y rules
     // https://github.com/evcohen/eslint-plugin-jsx-a11y/
     rules: {
+
+        // Enforce emojis are wrapped in and provide screenreader access.
+        // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/accessible-emoji.md
+        'jsx-a11y/accessible-emoji': 'error',
+
+        // Enforce all anchors to contain accessible content.
+        // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-has-content.md
+        'jsx-a11y/anchor-has-content': 'error',
+
+        // Enforce a clickable non-interactive element has at least one keyboard event listener.
+        // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md
+        'jsx-a11y/click-events-have-key-events': 'error',
+
+        // Enforce heading (h1, h2, etc) elements contain accessible content.
+        // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/heading-has-content.md
+        'jsx-a11y/heading-has-content': 'error',
+
+        // Enforce <html> element has lang prop.
+        // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/html-has-lang.md
+        'jsx-a11y/html-has-lang': 'warn',
```

Comment:
not sure how useful this is given that we'll never have `<html>` in a react component since it has to sit inside of a node on a page